data_steward/validation/main.py

```diff
@@ -696,8 +697,10 @@ def perform_validation_on_file(file_name, found_file_names, hpo_id,
 
 
 def _validation_done(bucket, folder):
-    if gcs_utils.get_metadata(bucket=bucket,
-                              name=folder + common.PROCESSED_TXT) is not None:
+    storage_client = StorageClient()
+    sc_bucket = storage_client.get_bucket(bucket)
```

Comment:
I just wanted to mention that this is fine in this case, but instantiating a client is resource intensive. Ideally we'd want to pass in already created clients to the function or use their built-ins/added functionalities to achieve this. Maybe in the future a refactor would require the caller of this function to pass the client object which can be used here.

Reply:
Yes. I thought about that and decided to do it like this since it is just one method that is using it but will have only one client for all methods to call as more of them need to be modified in the future